pycoin/networks/registry.py

```diff
@@ -22,7 +22,7 @@ def network_for_netcode(symbol):
                 module.symbol = symbol
                 return module.network
         except (AttributeError, ImportError):
-            pass
+            return
```

Comment:
I don't think you want to do this. It prevents all subsequent networks from loading, which is bad.
```suggestion
            pass
```